extensions/ezmsg-sigproc/ezmsg/sigproc/ewmfilter.py

```diff
@@ -73,9 +73,12 @@ async def sync_output(self) -> AsyncGenerator:
             buffer_data = buffer.data
             buffer_data = np.moveaxis(buffer_data, axis_idx, 0)
 
+            while scale_arr.ndim < buffer_data.ndim:
```

Comment:
Could also change this to
```python
if scale_arr.ndim < buffer_data.ndim:
    scale_arr = np.expand_dims(scale_arr, list(range(scale_arr.ndim, buffer_data.ndim)))
```

Reply:
Definitely happy to get rid of a while loop!
Let me know if you're interested in merging this PR in theory then I'll make the requested edits.

Reply:
It all looks good to me. I'll defer to @griffinmilsap if he'd really like to get some tests in place prior to making changes.